CHANGELOG.md

```diff
@@ -20,6 +20,8 @@
 
 ### Internal
 
+- Upgrade razzle to `^3.1.5`. @tiberiuichim
```

Comment:
Just wondering here if that's "internal". Need to think about it and maybe do some research...